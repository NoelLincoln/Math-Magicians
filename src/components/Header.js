import { NavLink } from 'react-router-dom';
import styles from '../styles/header.css';

const Header = () => (
  <>
    <nav className={styles.nav}>
      <h1>Math Magicians</h1>
      <ul>
        <li>
          <NavLink to="/">Home</NavLink>
        </li>
        <li>
          <NavLink to="/calculator">Calculator</NavLink>
        </li>
        <li>
          <NavLink to="/quotes">Quotes</NavLink>
        </li>
      </ul>
    </nav>
  </>
);

export default Header;
