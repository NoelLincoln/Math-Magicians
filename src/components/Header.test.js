import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom'; // Import BrowserRouter
import '@testing-library/jest-dom';
import Header from './Header';

test('Header component snapshot', () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Header />
    </BrowserRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
