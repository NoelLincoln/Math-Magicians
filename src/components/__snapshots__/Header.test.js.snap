// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapshot 1`] = `
<DocumentFragment>
  <nav>
    <h1>
      Math Magicians
    </h1>
    <ul>
      <li>
        <a
          aria-current="page"
          class="active"
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          class=""
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          class=""
          href="/quotes"
        >
          Quotes
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
